client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -139,7 +139,11 @@ class Upload extends React.Component {
 
 		// DropZone supplies an array, FilePicker supplies a FileList
 		const file = files[ 0 ] || files.item( 0 );
-		if ( file.type !== 'application/zip' ) {
+
+		const validFileType = file.type === 'application/zip' || file.type === 'application/x-zip-compressed';
```

Comment:
Interested in the reason for the addition of the `const validFileType` here rather than keeping the check inside the `if` below?